.golangci.toml

```diff
@@ -0,0 +1,42 @@
+[run]
```

Comment:
Is it possible to have the config in yaml format?

Reply:
done :-)